package.json

```diff
@@ -11,7 +11,10 @@
 	"type": "module",
 	"exports": {
 		".": {
-			"types": "./lib/index.d.ts",
+			"types": {
+				"import": "./lib/index.d.ts",
+				"require": "./lib/index.d.cts"
+			},
```

Comment:
FYI, you could remove this whole block and it would still work, since the types are happily sharing the same directory as their JS partners and have appropriate file extensions. Some people still like to explicitly list `types` in the exports, and that’s fine too.